pl_bolts/models/detection/yolo/__init__.py

```diff
@@ -0,0 +1,4 @@
+from pl_bolts.models.detection.yolo.yolo_config import YoloConfiguration
+from pl_bolts.models.detection.yolo.yolo_module import Yolo
+
+__all__ = ["YoloConfiguration", "Yolo"]
```

Comment:
Could we make this file empty? Otherwise, we'll have two ways to import these names:
```python
from pl_bolts.models.detection import Yolo       # <- This is fine.
from pl_bolts.models.detection.yolo import Yolo  # <- We don't want this to keep the API simple. (I think)
```

Reply:
I think we can just rename yolo.py `yolo_module` like we have done for other detection models

Reply:
@oke-aditya there's no yolo.py. There's a yolo subdirectory, like faster_rcnn.  @akihironitta FasterRCNN is also imported in both [detection/__init__.py](https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/master/pl_bolts/models/detection/__init__.py) and [detection/faster_rcnn/__init__.py](https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/master/pl_bolts/models/detection/faster_rcnn/__init__.py). Should I clear `detection/faster_rcnn/__init__.py` too, or leave them like this?

Reply:
Nice catch. I will solve faster RCNN in Retina Net PR. This will avoid merge conflicts too. 
Hopefully I will get retina Net completed by this week. (Really sorry for lack of time)

Reply:
> @akihironitta FasterRCNN is also imported in both [detection/__init__.py](https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/master/pl_bolts/models/detection/__init__.py) and [detection/faster_rcnn/__init__.py](https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/master/pl_bolts/models/detection/faster_rcnn/__init__.py). Should I clear `detection/faster_rcnn/__init__.py` too, or leave them like this?

@senarvi There are similar structures in `pl_bolts/models/*` as you point out, and we are actually trying to refactor these as discussed in https://github.com/PyTorchLightning/pytorch-lightning-bolts/issues/427#issuecomment-762032327, so let's make this file empty.

Reply:
Fixed.